examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Startup.cs

```diff
@@ -22,6 +23,11 @@ public void ConfigureServices(IServiceCollection services)
 
             // We add a Settings model to the service container, which takes its values from the applications configuration.
             services.Configure<Settings>(Configuration.GetSection("WebDemo:Settings"));
+
+            services.AddAzureAppConfiguration();
```

Comment:
Please add a comment

Reply:
Added comment from the package.

---
In reply to: [464787723](https://github.com/Azure/AppConfiguration/pull/329#discussion_r464787723) [](ancestors = 464787723)